src/Ushahidi/Modules/V5/Repository/TagRepository.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Repository;
+
+use Ushahidi\Core\Entity\Tag;
+use Illuminate\Support\Collection;
+use Ushahidi\Core\EloquentRepository;
+use Ushahidi\Contracts\Repository\Entity\TagRepository as TagRepositoryInterface;
+
+class TagRepository extends EloquentRepository implements TagRepositoryInterface
+{
+    protected static $root = Tag::class;
+
+    public function doesTagExist($value)
+    {
+    }
```

Comment:
What do we need it for if its empty? If it's required by some interface/abstract class then it means we have to review that interface/abstract class.